packages/blockchain-wallet-v4-frontend/src/scenes/Transactions/selectors.ts

```diff
@@ -126,9 +128,15 @@ export const getData = (state, coin, isCoinErc20) =>
           : []
 
       return {
-        coinModel: coinModelR.getOrElse({}),
+        coinModel: coinModelR.getOrElse(
+          {} as <P extends WalletCurrencyType>(
+            p: P
+          ) => SupportedWalletCurrenciesType[P]
+        ),
         currency: currencyR.getOrElse(''),
         hasTxResults: !all(empty)(filteredPages),
+        supportedCoins: selectors.core.walletOptions.getSupportedCoins(state),
+        supportedCoinsR,
```

Comment:
dont return the `Remote` type (`supportedCoinsR`). just need the value which is the line above